addons/website/static/src/js/tours/specific_website_editor.js

```diff
@@ -0,0 +1,25 @@
+odoo.define('website.tour.specific_website_editor', function (require) {
+'use strict';
+
+var tour = require('web_tour.tour');
+
+tour.register('generic_website_editor', {
+    test: true,
+}, [{
+    trigger: 'a[data-action=edit]',
+    content: 'Click edit button',
+}, {
+    trigger: 'body:not([data-hello="world"])',
+    content: 'Check that the editor DOM matches its website-generic features',
```

Comment:
`run: function () {}, // it's a check`?

Reply:
Mmh, sure :+1: Actually I also have to check that the editor is loaded before checking that otherwise the first test is probably always true :p I'll change and squash with the fix.